test/configuration_test.dart

```diff
@@ -1,5 +1,4 @@
 @TestOn('browser')
-
```

Comment:
Please re-add the empty line here so it doesn't look like `@TestOn` refers to the first import.